.github/workflows/cuda.yml

```diff
@@ -26,10 +26,13 @@ jobs:
         include:
           - method: source
             python_version: 3.6
+            cuda_version: "11.2.0"
           - method: pip
             python_version: 3.7
+            cuda_version: "10.0"
```

Comment:
GitHub Actions treats values written as `x.0` like `int`s, so we need explicitly set them to strings.

Reply:
:scream: that's horrible haha